### ISSUE TITLE
eventbrite embed `<iframe>` does not include a title which fails Lighthouse accessibility report

### ISSUE THREAD

**Issue by thescientist13** [AnalogStudiosRI/www.blissfestri.com#76]
## Summary
When testing the #51 on Desktop in Lighthouse, the Tickets page was getting an Accessibility report for not having a `title` attribute on the `<iframe>` on the page, coming from the EventBrite embed.
https://dequeuniversity.com/rules/axe/4.6/frame-title
![Screen Shot 2023-05-14 at 5 31 10 PM](https://github.com/AnalogStudiosRI/www.blissfestri.com/assets/895923/46e5cd49-a686-424d-8015-333711a15e8a)

## Details
This is the EventBrite embed we have on the page, so it would be good to see if there is a way to fix this.  Here are some docs / links that might be helpful
- https://www.eventbrite.com/support/articles/en_US/Multi_Group_How_To/how-to-sell-eventbrite-tickets-on-your-website-through-an-embedded-checkout?lg=en_US
- 
- https://www.eventbrite.com/engineering/using-the-iframe-tag-to-help-keep-users-safe/
